stable/ceph-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{template "fullname" .}}
```

Comment:
Use consistent formatting throughout the chart:
Space after `{{` and before `}}`.